packages/smooth_app/lib/pages/hunger_games/question_page.dart

```diff
@@ -367,7 +274,7 @@ class _LoadingQuestionsView extends StatelessWidget {
           child: Center(
             child: DefaultTextStyle(
               textAlign: TextAlign.center,
-              style: const TextStyle(),
+              style: TextStyle(color: colorScheme.onSurface),
```

Comment:
Please
* make a screenshot
* remove all your color and style changes, and then make a screenshot